scipy/signal/tests/test_filter_design.py

```diff
@@ -3830,23 +3830,21 @@ class TestGammatone(object):
     def test_invalid_input(self):
         # Cutoff frequency is <= 0 or >= fs / 2.
         fs = 16000
-        with pytest.raises(ValueError, match='The frequency must be between '):
-            gammatone(-fs, 'iir', fs=fs)
-            gammatone(0, 'fir', fs=fs)
-            gammatone(fs / 2, 'iir', fs=fs)
-            gammatone(fs, 'fir', fs=fs)
+        for args in [(-fs, 'iir'), (0, 'fir'), (fs / 2, 'iir'), (fs, 'fir')]:
+            with pytest.raises(ValueError, match='The frequency must be '
+                               'between '):
+                gammatone(*args, fs=fs)
 
         # Filter type is not fir or iir
-        with pytest.raises(ValueError, match='ftype must be '):
-            gammatone(440, ftype='fie', fs=16000)
-            gammatone(220, ftype='it', fs=32000)
+        for args in [(440, 'fie'), (220, 'it')]:
+            with pytest.raises(ValueError, match='ftype must be '):
+                gammatone(*args, fs=fs)
```

Comment:
Unless I'm mistaken it looks like this test has a changed meaning because `fs` is now the same for both `gammatone()` incantations?

For catching an exception it may not matter too much I suppose.

Reply:
others tests exercise `fs=1000,16000,24000` which feels sufficient for me (given this test isn't fundementally about fs)